provider/resource_collection.go

```diff
@@ -531,44 +661,6 @@ func collectionApiEntriesDeleteOldRows(ctx context.Context, d *schema.ResourceDa
 // Collection Resource
 // ----------------------------------------------------------------------
 
-func collectionCreate(ctx context.Context, d *schema.ResourceData, m interface{}) (diags diag.Diagnostics) {
```

Comment:
same as the update one, but a bit different logging. But we know from the plan which function exactly will be called